tests/unit/tasks/server/middleware/tests-server-test.js

```diff
@@ -20,6 +20,16 @@ describe('TestServerAddon', function () {
       }
     };
 
+    afterEach(function() {
+      nextWasCalled = false;
+      mockRequest = {
+        method: 'GET',
+        path: '',
+        url: 'http://example.com',
+        headers: {}
+      };
+    });
```

Comment:
I think beforeEach is the right way to go here. Would be awesome if you could change it.